tasks/manage_app.yml

```diff
@@ -14,7 +14,7 @@
 
 - name: Set OpenShift objects to manage
   set_fact:
-    deployments: "{{ templates | map('regex_search', '.*/dc*.yml.j2$') | select('string') | list }}"
+    deployments: "{{ templates | map('regex_search', '.*/dc.*.yml.j2$') | select('string') | list }}"
```

Comment:
will it take dc.yml.j2?

Reply:
Yes `.*` means any occurrence (of any character) even none.